src/components/refactor/utils/utils.js

```diff
@@ -69,3 +69,12 @@ export const areNumbersOrdered = (array) => {
     }
     return true;
 };
+
+export const findIndexesOfDuplicateFieldValues = (values, fieldName) => {
```

Comment:
I'm a bit confused with how this function should be used, I think we should not need "fieldName" to check duplicates, it would make the function easier to reuse

Reply:
I find that this function is quite generic. The function returns an array of the indices of the elements in values for which the value of the fieldName property is duplicated.